CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to Fleet
+
+Fleet accepts contributions via GitHub issues and pull requests.
+This document outlines the process to get your pull request accepted.
+
+Fleet does accept external contributions in general, however the teams time is limited. Fleet is on a quarterly release cycle, so it might take a long time for external contributions to be commented on, let alone reviewed and merged.
```

Comment:
```suggestion
Fleet does accept external contributions in general, however the team's time is limited. Fleet is on a quarterly release cycle, so it might take a long time for external contributions to be commented on, let alone reviewed and merged.
```

Reply:
Maybe add a section like

"We're working on improving review times in the future"